modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -251,3 +251,10 @@
                               :filter_values       (field->parameter-value query)})
        " */ "
        (qputil/default-query->remark query)))
+
+(defmethod sql-jdbc.sync/have-any-schema-privileges? :redshift [_ ^Connection conn ^String table-schema]
```

Comment:
This isn't in the Clojure style guide but generally I think it looks nicer to have the args on a new line for `defmethod` forms -- that's the convention we've been using in most of the MB code

Reply:
No longer relevant with latest change.